resources/styles/global/forms.less

```diff
@@ -33,6 +38,27 @@ textarea {
   }
 }
 
+.invalid-form {
```

Comment:
Sure, I'll do that now.

On Wed, Jun 10, 2015 at 11:19 PM, Philip Schatz <notifications@github.com>
wrote:

> In resources/styles/global/forms.less
> <https://github.com/openstax/tutor-js/pull/384#discussion_r32128802>:
>
> > @@ -33,6 +38,27 @@ textarea {
> >    }
> >  }
> >
> > +.invalid-form {
>
> Hmm, what about is- prefixes for classes conveying boolean states? (like
> is-valid, is-required, and is-empty?)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openstax/tutor-js/pull/384/files#r32128802>.
>